docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
+
+pushd %~dp0
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
```

Comment:
Add document about if developer want to test doc changes. It should support both Linux and Windows, and it's better the same commands.